### PR TITLE
Reorganizing some bits in the App Shop

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/69213

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$product' | [app-shop.php:74](https://github.com/moderntribe/tribe-common/blob/0270fdab2b7a7a82f631f551c6b9d9dccafbfb3c/src/admin-views/app-shop.php#L74)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$button_label' | [app-shop.php:84](https://github.com/moderntribe/tribe-common/blob/0270fdab2b7a7a82f631f551c6b9d9dccafbfb3c/src/admin-views/app-shop.php#L84)

> via jenkins/codesniffer-phpcs

File: src/admin-views/app-shop.php [hunk 1]
```diff
@@ -4,39 +4,94 @@
...
 						</div>
+						<div class="caption">
+							<h4><a href="<?php echo esc_url( $product->link ); ?>"><?php echo esc_html( $product->title ); ?></a></h4>
+
+							<div class="description">
+								<p><?php echo $product->description; ?></p>
+								<?php
+								if ( isset( $product->requires ) ) {
+									?>
+									<p><strong><?php esc_html_e( 'Requires:', 'tribe-common' );?></strong> <?php echo esc_html( $product->requires );?></p>
```
Comment (barryhughes) [inline src/admin-views/app-shop.php]: Space after the final `;` and before the closing `?>`

File: src/admin-views/app-shop.php [hunk 2]
```diff
@@ -4,39 +4,94 @@
...
+					case 4:
+					case 7:
+					default:
+						$wide_indexes = array( 0 );
+				}
+
+				$i = 0;
+				foreach ( $products as $product ) {
+					?>
+					<div class="tribe-addon<?php echo in_array( $i, $wide_indexes ) ? ' first' : '';?>">
```
Comment (barryhughes) [inline src/admin-views/app-shop.php]: Space after the final `;` and before the closing `?>`

File: src/Tribe/App_Shop.php [hunk 3]
```diff
@@ -105,6 +105,7 @@ private function get_all_products() {
 					'link' => 'https://theeventscalendar.com/product/event-aggregator/?utm_campaign=in-app&utm_source=addonspage&utm_medium=event-aggregator&utm_content=appstoreembedded-1',
 					'description' => __( 'Importing events from multiple sources has never been easier! Event Aggregator helps you curate and manage event import feeds from Facebook, Meetup, Google Calendar, iCalendar, CSV, and ICS. Schedule automatic imports or manually import events when you’re ready. Event Aggregator provides a convenient dashboard to manage bulk imports, filters, one-way sync, import history, and more.', 'tribe-common' ),
 					'image' => 'images/app-shop-ical.jpg',
+					'is_installed' => tribe( 'events-aggregator.main' )->is_service_active(),
```
Comment (zbtirrell) [inline src/Tribe/App_Shop.php]: We can't do this until 4.4...

**Pull request merged**